sphinx/screens/chats/chat-contact/chat-contact/src/main/java/chat/sphinx/chat_contact/ui/ChatContactViewModel.kt

```diff
@@ -56,7 +57,7 @@ internal class ChatContactViewModel @Inject constructor(
     app: Application,
     dispatchers: CoroutineDispatchers,
     memeServerTokenHandler: MemeServerTokenHandler,
-    chatNavigator: ContactChatNavigator,
+    private val contactChatNavigator: ContactChatNavigator,
```

Comment:
remove `private val` as the `chatNavigator` is accessible from the `ChatViewModel` abstraction and can be cast as the `ContactChatNavigator` safely.